tests/queries/0_stateless/01710_projection_aggregation_in_order.sql

```diff
@@ -31,8 +31,11 @@ SET allow_experimental_projection_optimization=1, optimize_aggregation_in_order=
 WITH toStartOfHour(ts) AS a SELECT sum(value) v FROM normal WHERE ts > '2021-12-06 22:00:00' GROUP BY a ORDER BY v LIMIT 5;
 WITH toStartOfHour(ts) AS a SELECT sum(value) v FROM normal WHERE ts > '2021-12-06 22:00:00' GROUP BY toStartOfHour(ts), a ORDER BY v LIMIT 5;
 
+<<<<<<< HEAD
```

Comment:
clean this. if we have no time to run this stateless let's post it in tests/queries_not_supported folder.
proton-client supports `sql` file input.
it is better to manually run the statless tests to make sure we do not break the compatibility in later code changes.